### PR TITLE
feat: EIP1559 Custom fee

### PR DESCRIPTION
**Pull request by AndonMitev**
#Linear Ticket

### CODE REVIEW

File: src/components/CustomFeesEIP1559.vue [hunk 1]
```diff
@@ -0,0 +1,633 @@
...
+      return isNaN(fiat) ? 0 : fiat
+    },
+    maxFiat () {
+      const fiat = prettyFiatBalance(
+        getSendFee(this.nativeAsset, this.maxFee),
+        this.fiatRates[this.nativeAsset]
+      )
+      return isNaN(fiat) ? 0 : fiat
+    },
+    minimum() {
```
Comment (beemi) [inline src/components/CustomFeesEIP1559.vue:236]: extra space, lint issue

File: src/components/CustomFeesEIP1559.vue [hunk 2]
```diff
@@ -0,0 +1,633 @@
...
+      const totalMinFee = getSendFee(this.nativeAsset, this.baseFee).plus(this.totalFees.slow)
+      return {
+        amount: totalMinFee,
+        fiat: prettyFiatBalance(
+          this.totalFees.slow,
+          this.fiatRates[this.nativeAsset]
+        )
```
Comment (beemi) [inline src/components/CustomFeesEIP1559.vue:243]: extra space, lint issue

File: src/components/CustomFeesEIP1559.vue [hunk 3]
```diff
@@ -0,0 +1,633 @@
...
+      const totalMinFee = getSendFee(this.nativeAsset, this.baseFee).plus(this.totalFees.slow)
+      return {
+        amount: totalMinFee,
+        fiat: prettyFiatBalance(
+          this.totalFees.slow,
+          this.fiatRates[this.nativeAsset]
+        ) 
+      }
+    },
+    maximum() {
```
Comment (beemi) [inline src/components/CustomFeesEIP1559.vue:246]: space after function start, lint issue

**Review by beemi — APPROVED** [review]
please run `npm run lint`

**Review by harshjv — CHANGES_REQUESTED** [review]
Few comments;

1. To enable EIP1559, you need to pass `hardfork = 'london'` to JsWallet provider. You can change `createEthereumClient` and accept `hardfork` argument.
2. Fee provider needs to be added immediately after Rpc provider. Can you move `ethClient.addProvider(feeProvider)` below Rpc provider? https://github.com/liquality/wallet/blob/f3249c005e8ee3631917da351aa3a03f43a66c3f/src/store/factory/client.js#L129 below Rpc provider?
3. Polygon testnet supports EIP1559. Can you enable it here? https://github.com/liquality/wallet/blob/f3249c005e8ee3631917da351aa3a03f43a66c3f/src/store/factory/client.js#L206

**Comment by AndonMitev** [comment]
> Few comments;
> 
> 1. To enable EIP1559, you need to pass `hardfork = 'london'` to JsWallet provider. You can change `createEthereumClient` and accept `hardfork` argument.
> 2. Fee provider needs to be added immediately after Rpc provider. Can you move `ethClient.addProvider(feeProvider)` below Rpc provider? https://github.com/liquality/wallet/blob/f3249c005e8ee3631917da351aa3a03f43a66c3f/src/store/factory/client.js#L129
>     below Rpc provider?
> 3. Polygon testnet supports EIP1559. Can you enable it here? https://github.com/liquality/wallet/blob/f3249c005e8ee3631917da351aa3a03f43a66c3f/src/store/factory/client.js#L206

done :)

**Review by harshjv — APPROVED** [review]
LGTM

**Pull request merged**